package.json

```diff
@@ -7,11 +7,12 @@
   "author": "Carto Dev Team",
   "license": "BSD-3-Clause",
   "devDependencies": {
-    "exquisite-sst": "^1.4.0",
+    "chalk": "^2.4.1",
```

Comment:
we are using chalk and colors.js, we should chose one to reduce install time :)

Reply:
Oops! I didn't realize, let me change it!